extensions/data-transfer/portability-data-transfer-apple/src/main/java/org/datatransferproject/datatransfer/apple/photos/AppleMediaInterface.java

```diff
@@ -267,29 +272,31 @@ private String sendPostRequest(@NotNull String url, @NotNull final byte[] reques
         AuditKeys.errorCode, con.getResponseCode(),
       e);
 
-      throw new AppleHttpException(e.getMessage(), e, con.getResponseCode());
+      convertAndThrowException(e, con);
     } finally {
       con.disconnect();
     }
     return responseString;
   }
 
-  private void convertAndThrowException(@NotNull final AppleHttpException e)
-      throws CopyExceptionWithFailureReason {
+  private void convertAndThrowException(@NotNull final IOException e, @NotNull final HttpURLConnection con)
```

Comment:
Q: Should any of these cases not throw errors and just be skipped instead? I think becuase they are CopyExceptions they will cause the job to automatically fail, but if you convert them into an IOException or a different exception they will get retried using the executor (and within Google we have the skippable errors in place so we can tag the exception to just skip instead of retry)

Reply:
Thank, that makes sense, I have updated some errors to be IOException to make it retryable and skippable.